api/v1beta1/hosted_controlplane.go

```diff
@@ -211,16 +219,15 @@ type HostedControlPlaneStatus struct {
 	// +kubebuilder:validation:Optional
 	OAuthCallbackURLTemplate string `json:"oauthCallbackURLTemplate,omitempty"`
 
-	// Version is the semantic version of the release applied by
-	// the hosted control plane operator
-	// +kubebuilder:validation:Optional
-	Version string `json:"version,omitempty"`
-
-	// ReleaseImage is the release image applied to the hosted control plane.
-	ReleaseImage string `json:"releaseImage,omitempty"`
+	// Version is the status of the release version applied by the
+	// hosted control plane operator.
+	// +optional
+	Version *ClusterVersionStatus `json:"version,omitempty"`
```

Comment:
FIXME: If we're deprecating the existing string `Version` but keeping it around, what do we call this property?

Reply:
I am wondering if this is going to be acceptable level for change for hypershift  cc @csrwng

Reply:
We need to call it something other than `Version` and keep the previous field. 
How about `VersionStatus` ?

Reply:
I've  moved to `VersionStatus` in d2909321.